src/dfx/src/commands/canister/create.rs

```diff
@@ -80,6 +82,23 @@ pub struct CanisterCreateOpts {
     /// Bypasses the Wallet canister.
     #[arg(long)]
     no_wallet: bool,
+
+    /// Transaction timestamp, in nanoseconds, for use in controlling transaction deduplication, default is system time.
+    /// https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/#transaction-deduplication-
+    //TODO(SDK-1331): unhide
+    #[arg(long, hide = true)]
+    created_at_time: Option<u64>,
```

Comment:
The `--created-at-time` and `--all` options should conflict with each other, because `dfx canister create --all --created-at-time <t>` could try to assign the same canister id to more than one canister.